sdk/eventhub/Azure.Messaging.EventHubs/samples/Infrastructure/ISample.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.EventHubs.Samples.Infrastructure
+{
+    /// <summary>
+    ///   Designates a class as a sample
+    /// </summary>
+    ///
+    public interface ISample
```

Comment:
Yes!   That little tweak seems to make things a lot more clear when looking at the interface.   Nicely done!